wrapper/wrapper.go

```diff
@@ -50,10 +56,101 @@ type Specification struct {
 
 // Statistics holds all available statistics: network, tcp/tcp6, cgroups and filesystem
 type Statistics struct {
-	Network     []NetworkInterface             `json:"network"`
-	Connection  TcpInterface                   `json:"connection"` //TCP, TCP6 connection stats
-	CgroupStats *cgroups.Stats                 `json:"cgroups"`
-	Filesystem  map[string]FilesystemInterface `json:"filesystem"`
+	Network         []NetworkInterface             `json:"network"`
+	Connection      TcpInterface                   `json:"connection"` //TCP, TCP6 connection stats
+	CgroupStats     *cgroups.Stats                 `json:"cgroups"`
+	CgroupsExtended *CgroupExtended                `json:"cgroups_extended"`
+	Filesystem      map[string]FilesystemInterface `json:"filesystem"`
+}
+
+// CgroupsExtended holds additional group statsistics like cpuset and shares
+type CgroupExtended struct {
+	Cpuset CpuSet `json:"cpuset"`
+	Shares Shares `json:"shares"`
+}
+
+// GetExtendedStats extracts CPUs and memory nodes a group can access
+func (cs *CpuSet) GetExtendedStats(path string, ext *CgroupExtended) error {
+	cpus, err := ioutil.ReadFile(filepath.Join(path, "cpuset.cpus"))
```

Comment:
Should we consider situation when these files are unavailable. I imagine, that you can start docker without cpu-sets (or am I wrong?)